packages/experiments/src/components/Stack/Stack.types.ts

```diff
@@ -2,27 +2,96 @@ import { IStyle } from '../../Styling';
 import { IStyleableComponent } from '../../Foundation';
 
 export interface IStackProps extends IStyleableComponent<IStackProps, IStackStyles> {
+  /**
+   * How to render the Stack.
+   */
   renderAs?: string | React.ReactType<IStackProps>;
+
+  /**
+   * CSS class name used to style the Stack.
+   */
   className?: string;
 
-  fill?: boolean;
-  collapseItems?: boolean;
+  /**
+   * Whether to render Stack child elements horizontally.
+   */
+  horizontal?: boolean;
+
+  /**
+   * How to align Stack child elements horizontally (along the x-axis).
+   */
+  horizontalAlignment?:
+    | 'start'
+    | 'end'
+    | 'center'
+    | 'space-between'
+    | 'space-around'
+    | 'space-evenly'
+    | 'baseline'
+    | 'stretch';
+
+  /**
+   * How to align Stack child elements vertically (along the y-axis).
+   */
+  verticalAlignment?:
+    | 'start'
+    | 'end'
+    | 'center'
+    | 'space-between'
+    | 'space-around'
+    | 'space-evenly'
+    | 'baseline'
+    | 'stretch';
+
+  /**
+   * Whether the Stack should take up 100% of the width of its parent.
+   */
+  fillHorizontal?: boolean;
 
-  inline?: boolean;
-  vertical?: boolean;
+  /**
+   * Whether the Stack should take up 100% of the height of its parent.
+   */
+  fillVertical?: boolean;
 
+  /**
+   * Whether Stack child elements should shrink to fit the available space.
+   */
+  collapseItems?: boolean;
```

Comment:
Hmm, shrink and collapse (collapsing as in hiding elements) are different concepts to me. Should this be called shrinkItems instead?

Reply:
`shrinkItems` sounds good

Reply:
To me, shrinkToFit implies that the entire Stack shrinks to fit the available space, rather than applying flex-shrink to each of the Stack's children.

Reply:
I am wondering what the default behavior should be, shouldn't it be shrinktofit? maybe we should name this the opposite of the default.